components/brave_wallet/browser/android_page_appearing_browsertest.cc

```diff
@@ -394,6 +394,7 @@ IN_PROC_BROWSER_TEST_F(AndroidPageAppearingBrowserTest,
     const std::vector<std::string> ignore_patterns = {
         "TypeError: Cannot read properties of undefined (reading "
         "'onCompleteReset')",
+        "Error: An internal error has occurred",
```

Comment:
Why is that needed now?

Reply:
Not sure it is yet, attempting to debug repeat failing `Android` test